instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/yaml/JmxConfig.java

```diff
@@ -20,22 +20,22 @@ public class JmxConfig {
   //     - JMX_DEFINITION1
   //     - JMX_DEFINITION2
   // The parser is guaranteed to call setRules with a non-null argument, or throw an exception
```

Comment:
Please remove this comment line as well, no longer applicable.

Reply:
Done. Thanks!